neil

```diff
@@ -289,6 +325,19 @@
         s (str (str/trim (str nodes)) "\n")]
     (spit (:deps-file opts) s)))
 
+(defn dep-versions [opts]
+  (let [lib (or (:lib opts)
+                (first (:cmds opts)))
+        lib (symbol lib)
+        git? (or (:sha opts)
+                 (:latest-sha opts))
+        versions (if git?
```

Comment:
The current git logic makes no sense.

If we decide _not_ to support Git initially, we could print nothing (empty list of versions) or fail. But one would never call the versions subcommand _and_ specify a version.

Reply:
Wrote proper error handling. User is notified on stderr if coordinates are not found on Maven or Clojars.